src/main/java/org/jenkinsci/plugins/workflow/graphanalysis/NodeStepNamePredicate.java

```diff
@@ -63,11 +65,13 @@ public boolean apply(@Nullable FlowNode input) {
                 // Workaround for cases where someone is using the latest workflow-api which has StepNode
                 //  but manages to skip the post-2.26 versions of workflow-cps where it a parent of workflow-cps StepNode.
                 // Technically consumers *were* supposed to call ForkScanner#setParallelStartPredicate, but better to be foolproof.
-                Method getDescriptorMethod = input.getClass().getDeclaredMethod("getDescriptor", null);
+                Method getDescriptorMethod = input.getClass().getMethod("getDescriptor", null);
```

Comment:
IIRC you will sometimes get a warning with this syntax, because it is ambiguous whether you mean `(Class[]) null` or `new Class[] {null}`; the compiler should resolve to the former (the one you want) but it is better not to rely on that. If you do not want to use varargs, then the way to make that explicit is

```java
.getMethod("getDescriptor", new Class[0])
```